src/main/kotlin/no/nav/syfo/kafka/oppfolgingstilfelle/OppfolgingstilfelleKafkaConsumer.kt

```diff
@@ -49,16 +49,18 @@ class OppfolgingstilfelleKafkaConsumer(
                 try {
                     val peker: KOppfolgingstilfellePeker = objectMapper.readValue(it.value())
                     val aktorId = peker.aktorId
+                    val orgnummer = peker.orgnummer
                     val fnr = accessControl.getFnrIfUserCanBeNotified(aktorId)
                     fnr?.let {
                         varselPlanners.forEach { planner ->
                             try {
-                                runBlocking { planner.processOppfolgingstilfelle(aktorId, fnr) }
+                                runBlocking { planner.processOppfolgingstilfelle(aktorId, fnr, orgnummer) }
                             } catch (e: Exception) {
                                 log.error("Error in [${planner.name}] planner: | ${e.message}", e)
                                 tellFeilIPlanner()
                             }
                         }
+                        log.info("Bruker med forespurt fnr kan ikke varsles")
                     }
```

Comment:
Jeg lurer på om denne logmeldingen er litt feil plassert? Hvis tanken var at den skal skrives ut når fnr er null, så bør det være
```suggestion     
                    } ?: log.info("Bruker med forespurt fnr kan ikke varsles")
```
Kanskje man også skulle skrevet noe om at det kan skyldes at de er gradert eller reservert?